src/Widgets/WindowCloneContainer.vala

```diff
@@ -361,24 +361,26 @@ namespace Gala {
 
             // make sure our windows are where they belong in case they were moved
             // while were closed.
-            foreach (var window in get_children ())
-                ((WindowClone) window).transition_to_original_state (false);
-
-            reflow ();
+            if (!is_cancel_animation) {
+                foreach (var window in get_children ())
+                    ((WindowClone) window).transition_to_original_state (false);
```

Comment:
elementary code style is to use brackets even for one-liners. It looks like this was already incorrect before you got here, but if we're touching this line anyways might as well fix it

```suggestion
                foreach (var window in get_children ()) {
                    ((WindowClone) window).transition_to_original_state (false);
                }
```